rct229/data_fns/table_G3_5_3_fns.py

```diff
@@ -43,19 +43,19 @@ def table_3_5_3_lookup(compressor_type, capacity):
 
     minimum_capacity = 0
```

Comment:
I may prefer another type of implementation, we can go over it tomorrow.

```python
# Make sure this line is a sorted (ascending order) list
capacity_threshold_list = [0, 150, 300, 9999.99]

...
function...

# this line converts the list to list of quantities.
capacity_threshold_list_ton = list(map(ct: ct*ureg("ton"), capacity_threshold_list))
minimum_capacity = min(capacity_threshold_list_ton)
maximum_capacity = max(capacity_threshold_list_ton)
for capacity_threshold_ton in capacity_threshold_list_ton:
    if capacity > capacity_threshold_ton :
        minimum_capacity = capacity_threshold_ton 
    if capacity < capacity_threshold_ton :
        maximum_capacity = capacity_threshold_ton 
        break
```

@charliepnnl  any better suggestion to make the code cleaner?

Reply:
Great point! I fixed based on Weili's comment. However, if there is a room to improve this code, I'll update this part during our meeting.